Build fail on GNU/Linux distros (Fedora) due to missing strnstr() function

cc -g -O -Wall -Werror  -DVERSION='"1.1.255"' -I/usr/include/libusb-1.0   -c -o radio.o radio.c
radio.c: In function ‘radio_validate_config’:
radio.c:322:9: error: implicit declaration of function ‘strnstr’; did you mean ‘strstr’? [-Werror=implicit-function-declaration]
  322 |     if (strnstr(line, "Radio:", strlen(line))) {
      |         ^~~~~~~
      |         strstr
cc1: all warnings being treated as errors
make: *** [<builtin>: radio.o] Error 1

--- a/radio.c
+++ b/radio.c
@@ -291,9 +291,9 @@ void radio_validate_config(const char *filename) {
 
   // First, find out which radio it is.
   while (fgets(line, sizeof(line), conf)) {
-    if (strnstr(line, "Radio:", strlen(line))) {
+    if (strstr(line, "Radio:")) {
       for (i = 0; radio_tab[i].ident; i++) {
-        if (strnstr(line, radio_tab[i].ident, strlen(line))) goto found;
+        if (strstr(line, radio_tab[i].ident)) goto found;
       }
     }
   }

